src/ensembl/package.json

```diff
@@ -44,6 +44,7 @@
     "react": "16.8.1",
     "react-cookie": "^3.0.8",
     "react-dom": "16.8.1",
+    "react-ga": "^2.5.7",
```

Comment:
could we install exact versions of packages (not with tildes)?
`npm install something --save-exact` writes the exact version to package.json

Reply:
I've now updated it to use an exact value.